src/unix/udp.c

```diff
@@ -797,6 +797,37 @@ int uv_udp_set_broadcast(uv_udp_t* handle, int on) {
 
 
 #if !defined(__NUTTX__)
+
+#if defined(__TIZENRT__)
+/*
+ * These flags are not supported for IPv6 by TizenRT
+ * The uv_set_socckopt expects both IPv6 and IPv4 versions of flag and checks
+ * internally, which flags should be used based on address family.
+ * This solution allows compilation of calls for TizenRT.
+ * An error for not supported flag will occur at runtime instead
+ * of compilation. It is better to be able to set TTL at least one protocol
+ * than not being able to set TTL at all for.
+ */
+static int get_ipv6_unicast_hops() {
+  return -1;
+}
+static int get_ipv6_multicast_hops() {
+  return -1;
+}
+static int get_ipv6_multicast_loop() {
+  return -1;
+}
+#else
+static int get_ipv6_unicast_hops() {
+  return IPV6_UNICAST_HOPS;
```

Comment:
https://github.com/Samsung/libtuv/pull/93 introduced these macros. Wouldn't be better to change them to `-1` at line 72:80?